velox/vector/FlatVector-inl.h

```diff
@@ -372,5 +372,19 @@ void FlatVector<T>::prepareForReuse() {
     rawValues_ = nullptr;
   }
 }
+
+template <typename T>
+void FlatVector<T>::setStringBuffers(std::vector<BufferPtr> buffers) {
```

Comment:
Specialize it for `StringView` then we can move it to cpp file

Reply:
It might be used for the other types?

Reply:
No string buffer is only useful when T = StringView